javascript/src/main/components/BasicCourseSearch/CourseFilters.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Form} from "react-bootstrap";
+
+
+const CourseFilters = ({cancelled, handleCancelledOnChange, closed, handleClosedOnChange, full, handleFullOnChange }) => {
+
+
+    return (
+        <Form.Group controlId="BasicSearch.Hide">
+            <Form.Check type="checkbox" label="Cancelled" value={cancelled} onChange={handleCancelledOnChange} id={`inline-checkbox-1`}/>
+            <Form.Check type="checkbox"  label="Closed" value={closed} onChange={handleClosedOnChange} id={`inline-checkbox-2`}/>
+            <Form.Check type="checkbox" label="Full" value={full} onChange={handleFullOnChange} id={`inline-checkbox-3`}/>
```

Comment:
If the value of `id` is a fixed string, I don't think we need to use JavaScript to populate it. I think the following should still work:

```suggestion
            <Form.Check type="checkbox" label="Full" value={full} onChange={handleFullOnChange} id="inline-checkbox-3"/>
```

Can you try it out and see if this still works as expected? If it does, we can replicate the change in the above two lines as well.

Reply:
Yes, it works with "".